packages/shared/getComponentName.js

```diff
@@ -9,13 +9,30 @@
 
 import type {Fiber} from 'react-reconciler/src/ReactFiber';
 
+import {
+  REACT_CALL_TYPE,
+  REACT_FRAGMENT_TYPE,
+  REACT_RETURN_TYPE,
+  REACT_PORTAL_TYPE,
+} from 'shared/ReactSymbols';
+
 function getComponentName(fiber: Fiber): string | null {
   const {type} = fiber;
+  if (typeof type === 'function') {
+    return type.displayName || type.name;
+  }
   if (typeof type === 'string') {
     return type;
   }
-  if (typeof type === 'function') {
-    return type.displayName || type.name;
+  switch (type) {
+    case REACT_FRAGMENT_TYPE:
+      return '<ReactFragment>';
```

Comment:
Should this just be `React.Fragment`? Same for others. Some our warnings put `<` and `/>` around names.